src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.ts

```diff
@@ -420,7 +438,45 @@ export class ServiceFormComponent extends CdForm implements OnInit {
         ]
       ],
       grafana_port: [null, [CdValidators.number(false)]],
-      grafana_admin_password: [null]
+      grafana_admin_password: [null],
+      // oauth2-proxy
+      https_address: [null, [
+        CdValidators.requiredIf({
+          service_type: 'oauth2-proxy'
```

Comment:
I think this is an optional field and we can move it below in that case

Reply:
Sure